index.html

```diff
@@ -3,184 +3,196 @@
-            <img src="https://assets.csh.rit.edu/pubsite/sponsors/oracle.png">
-            </a>
+<!-- Twitter embed -->
+
+<div style="background-color: #b0197e;" id="twitter_embed">
+    <h3 style="color: white; padding-top: 1em; margin-top: 0em;" class="left_offset">See what we've been up to ...</h3>
+    <div class="centered" style="padding-bottom: 1em; max-width: 90%; margin: auto;">
+        <a class="twitter-timeline" data-width="600" data-height="800" href="https://twitter.com/RITCSH?ref_src=twsrc%5Etfw" alt="Twitter feed. Please disable Firefox strict blocking to see this element. ">Tweets by RITCSH</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
+    </div>
+</div>
+
+<!-- Sponsors -->
+
+<div class="width-capped" style="background-color: #f4f4f6;">
+    <h3 class="left_offset" style="padding-top: 0.5em; margin-top: 0em; margin-bottom: 0em;">With a special thanks to our sponsors ...</h3>
+    <div class="container" id="my-homepage-sponsors">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/ritLarge.png" alt="RIT logo">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/google.png" alt="Google logo">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/microsoft.png" alt="Microsoft logo">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/slack.png" alt="Slack logo">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/hp.png" alt="HP Logo">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/oracle.png" alt="Oracle">
+    </div>
+</div>
+
+<!-- Banner above footer -->
+<div style="background-color: #1e1e1e;" id="gt511">
+    <div id="bottom_image">
+        <div id="promocont">
+            <img id="promo_logo"src="assets/images/bottom_banner_square.png"/>
         </div>
+        <div id="promo_center"> 
+            <h3 id="promo_text">Interested?<br>
+                <a href="https://www.csh.rit.edu/membership/intro-process.html" class="joinus">[Join us]</a>
+            </h3>                        
+        </div>  
     </div>
+</div>  
+
+<div style="background-color: #1e1e1e;" id="lt511">
+  <div id="promo_center"> 
+    <h3 id="promo_text">Interested?<br>
+      <a href="https://www.csh.rit.edu/membership/intro-process.html" class="joinus">[Join us]</a>
+    </h3>                        
+  </div> 
 </div>
+
+
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
```

Comment:
Don't we already pull these? https://github.com/ComputerScienceHouse/CSHPublicSite/blob/53647e071289f5da8a2823b6e388628dccff5045/_includes/head.html#L27-L29

Reply:
It does appear so, I commented them out and it still works.